slight/src/commands/run.rs

```diff
@@ -111,82 +134,163 @@ async fn shutdown_signal() {
 fn build_store_instance(
     toml: &TomlFile,
     toml_file_path: impl AsRef<Path>,
+    linked_capabilities: &mut HashSet<String>,
+    capability_store: &mut HashMap<String, BasicState>,
     resource_map: Arc<Mutex<StateTable>>,
     module: impl AsRef<Path>,
 ) -> Result<Builder> {
     let mut builder = Builder::new_default(module)?;
     let mut slight_builder = SlightCtxBuilder::default();
     builder.link_wasi()?;
-    if toml.specversion.as_ref().unwrap() == "0.1" {
+    if toml.specversion.as_ref().unwrap() == "0.2" {
```

Comment:
If we do not want to support backward compatibility, I would suggest we keep specversion at 0.1 for now?

Reply:
We do want to support backwards compatibility -- It's just not here yet. I was suggesting we could merge this as is, but, before triggering the `v0.2.0` release, make another PR TO ensure it's there. Do you think I should make it backwards compatible right off the gate?

Reply:
No, I imagine it will take some time to support the old specversion. I suggest we don't support the old one anymore, and keep specversion at `0.1`

Reply:
Actually, I just made it backwards compatible 😆